resources/hg-ReferralComposition.xml

```diff
@@ -1,534 +1,665 @@
+      </example>
+    </element>
+    <element id="Composition.section:core.section:proposedProcedure.code">
+      <path value="Composition.section.section.code" />
+      <min value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="240471000146100" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Composition.section:core.section:furtherImportant">
+      <path value="Composition.section.section" />
+      <sliceName value="furtherImportant" />
+      <max value="1" />
+    </element>
+    <element id="Composition.section:core.section:furtherImportant.extension">
+      <path value="Composition.section.section.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Composition.section:core.section:furtherImportant.extension:furtherImportantTextValue">
+      <path value="Composition.section.section.extension" />
+      <sliceName value="furtherImportantTextValue" />
+      <min value="1" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue" />
+      </type>
+    </element>
+    <element id="Composition.section:core.section:furtherImportant.extension:furtherImportantTextValue.value[x]">
+      <path value="Composition.section.section.extension.value[x]" />
+      <short value="FurtherImportant" />
+      <definition value="Geeft aan wat verder van belang is in het verwijsbericht, de updates en de eindrapportage.&#xD;&#xA;&#xD;&#xA;De HASP-richtlijn (NHG) geeft verschillende voorbeelden, zoals:&#xD;&#xA;* Eerder in 2015 speelden er surmenageklachten.&#xD;&#xA;* Lopend traject bij neuroloog vanwege spierklachten.&#xD;&#xA;* Op verzoek moeder." />
+      <alias value="VerderVanBelang" />
+      <min value="1" />
+      <mapping>
+        <identity value="ART-DECOR-hg-Kern" />
+        <map value="hg-dataelement-1713" />
+        <comment value="VerderVanBelang" />
+      </mapping>
+    </element>
+    <element id="Composition.section:core.section:furtherImportant.title">
+      <path value="Composition.section.section.title" />
+      <example>
+        <label value="Example for referral" />
+        <valueString value="Verder van belang" />
+      </example>
+    </element>
+    <element id="Composition.section:core.section:furtherImportant.code">
+      <path value="Composition.section.section.code" />
+      <min value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="159121000146100" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Composition.section:core.section:answerToYourQuestion">

```

Comment:
Isn't it a bit confusing that we're adding this element to the ReferralComposition profile, since this element is **only** allowed in the Update use case? I would add this element only to the UpdateComposition profile (that will be made in the future), or even better the generic Composition profile (that we'll create when tackling the profile layering).

Reply:
Alternatively, if it **is** relevant for the Referral use case, the data set in ART-DECOR should be changed accordingly.